### PR TITLE
Bug 797605 - RFE: Customer/Vendor/Employee Overview should also have …

### PR DESCRIPTION
**Pull request by christopherlam**
…"Process Payment" toolbar

image

### CODE REVIEW

**Comment by Bob-IT** [comment]
I have not pull this yet but noticed that the three objects have the same text so do you really need three duplicates or maybe you forgot to change some text?

File: gnucash/gnome/gnc-plugin-page-owner-tree.c [hunk 1]
```diff
@@ -314,6 +346,9 @@ static action_owners_struct action_owners[] =
     { "OTVendorReportAction",          GNC_OWNER_VENDOR },
     { "OTCustomerReportAction",        GNC_OWNER_CUSTOMER },
     { "OTEmployeeReportAction",        GNC_OWNER_EMPLOYEE },
+    { "OTProcessVendorPaymentAction",  GNC_OWNER_VENDOR },
+    { "OTProcessCustomerPaymentAction",GNC_OWNER_CUSTOMER },
+    { "OTProcessEmployeePaymentAction",GNC_OWNER_EMPLOYEE },
```
Comment (christopherlam) [inline gnucash/gnome/gnc-plugin-page-owner-tree.c:351]: @Bob-IT I think this is why... action for each type of owners

**Comment by Bob-IT** [comment]
OK I pulled this and if you are not requiring different text for each owner then it can just be one entry, just do not add any thing to the section above and use a general string for the action like 'OTProcessPaymentAction' unless I am missing something.

**Comment by christopherlam** [comment]
@Bob-IT Correct as usual :+1:

**Comment by jralls** [comment]
@christopherlam please revert this and apply it to master instead.

**Comment by christopherlam** [comment]
I'll wait till the next maint->master merge then apply to master.

**Comment by jralls** [comment]
Merged & pushed. Go for it.

**Pull request closed without merging**